### ISSUE TITLE
Thank you for your library

### ISSUE THREAD

**Issue by NistorCristian** [farzher/fuzzysort#46]
Hi,

This is not an issue. I just want to thank you @farzher for your amazing library. I just released a search library for Ghost Blogging Platform called [`ghost-search`](https://github.com/HauntedThemes/ghost-search) and it's using `fuzzysort` as a search algorithm. 

After a testing multiple libraries like `lunr`, `fuse`, and other search libraries, `fuzzysort` seems to be the one that does the job properly. When I started I was not expecting that finding a search library would be so hard. Boy I was wrong.

Thank you again and keep up the good work.

**Comment by d3lm** [comment]
Totally agreed! I have also tested a bunch of other libraries and this really does its job really well. Thanks for your work on this.

**Comment by WalterGR** [comment]
I agree.  I've been hacking on a local copy of https://github.com/minhanhhere/chrome-faster-bookmark (not my project; screenshots here: https://chrome.google.com/webstore/detail/faster-bookmark-add-bookm/fibadljlplamppipiebabkdehecpabce) which uses Fuse.  I'm getting much better results.  Thanks so much!

**Comment by dbuezas** [comment]
Plus one, I switched from fuse.js to fuzzysort in command palette extension for chrome (https://github.com/dbuezas/chrome-palette) and the search results are a lot more intuitive 🎉 

The only thing that is less good is that it is less of a stable search, meaning that the ordering of the input has less impact on the output.

In the extension I pre-sort commands by last used, and if the last one I used was `Split horizontally`, searching for `Split` still puts `Split vertically` first.

I assume the scoring function scores the latter higher because the match represents a higher percentage of the characters.

Anyway, great lib!
